style-dictionary/visual-refresh/metadata/colors.ts

```diff
@@ -609,6 +624,11 @@ const metadata: StyleDictionary.MetadataIndex = {
     public: true,
     themeable: true,
   },
+  colorTextToggleButtonIconPressed: {
+    description: 'The pressed text color of icon toggle buttons.',
```

Comment:
But toggle buttons of icon variant don't have text, do they? What is this token for?

Reply:
We normally use colorText tokens for icons' colors